roles/sro_deploy_custom_commit/tasks/deploy.yml

```diff
@@ -0,0 +1,25 @@
+- block:
+  - name: Git checkout
+    git:
+      repo: "{{ sro_git_repo }}"
+      dest: /tmp/special-resource-operator
+      version: "{{ sro_git_ref }}"
+      
+  - name: Deploy the custom version of the special resource operator
+    shell: |
+      TAG="{{ sro_image_tag }}" make deploy
```

Comment:
just a nit, but I think you can change that to:
```
command:
  make deploy TAG="{{ sro_image_tag }}"
```
if `TAG` is a `Makefile` variable